File: callback.js
const express = require('express');
const app = express();

app.use(express.json());

app.post('/callback', (req, res) => {
    console.log('Received data:', req.body);
    res.send('Data received');
});

app.listen(8080, '95.163.223.236', () => {
    console.log('Server is running on 95.163.223.236:8080');
});

File: callback.py
from flask import Flask, request

app = Flask(__name__)

@app.route('/callback', methods=['POST'])
def callback():
    data = request.json
    print(f"Received data: {data}")
    return "Data received", 200

if __name__ == '__main__':
    app.run(host='95.163.223.236', port=8080)